bindings/python/CHANGELOG.md

```diff
@@ -25,6 +25,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `ClientOptions::maxParallelApiRequests`;
 
+### Changed
+
+- Replaced `|` with `Union` type for Python 3.9 compatibility;
```

Comment:
Do you want to release this today?

Reply:
Yeah let's try, but also with https://github.com/iotaledger/iota-sdk/pull/1203 if possible since this will fix another thing

Reply:
Actually will split that one

Reply:
https://github.com/iotaledger/iota-sdk/pull/1210, so it's easier to review